### PR TITLE
Add comfortable remoteAddress change to FakeRequest

### PR DESCRIPTION
**Pull request by poslegm**

### CODE REVIEW

**Comment by gmethvin** [comment]
@poslegm Is it really worth the savings vs `copy(remoteAddress = address)`? And if we do this we'd want to do it consistently for all the other fields.

**Comment by poslegm** [comment]
@gmethvin `copy` method returns `RequestHeader` object. It is not comfortable. 
For creation `FakeHeader` object now used expression like `new FakeRequest(
        GET, "/", FakeHeaders(),
        AnyContentAsEmpty, remoteAddress = "127.0.0.1"
      )`. 
Maybe I do not know a more convenient way?

**Comment by gmethvin** [comment]
@poslegm Hmm. Yeah, it's a little bit weird since `FakeRequest` is a case class but doesn't define the copy method since `RequestHeader` does.

Why don't we add `with*` methods for all the all the fields (`uri`, `path`, `method`, `version`, et al.)?

Another possibility is to make the `_copy` method public and call it something else.

**Comment by poslegm** [comment]
@gmethvin I think that one copy method is better than methods for each field.

**Comment by typesafehub-validator** [comment]
Hi @poslegm,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

File: framework/src/play-test/src/main/scala/play/api/test/Fakes.scala [hunk 1]
```diff
@@ -39,7 +39,7 @@ case class FakeHeaders(data: Seq[(String, String)] = Seq.empty) extends Headers(
  */
 case class FakeRequest[A](method: String, uri: String, headers: Headers, body: A, remoteAddress: String = "127.0.0.1", version: String = "HTTP/1.1", id: Long = 666, tags: Map[String, String] = Map.empty[String, String], secure: Boolean = false) extends Request[A] {
 
-  private def _copy[B](
+  def changeFields[B](
```
Comment (gmethvin) [inline framework/src/play-test/src/main/scala/play/api/test/Fakes.scala]: Maybe just call it `copyFakeRequest` to distinguish it from the `copy` method in `RequestHeader`?

**Comment by poslegm** [comment]
@gmethvin Yes, this name is better suited

**Pull request merged**